backend.native/tests/build.gradle

```diff
@@ -3726,6 +3748,11 @@ linkTest("inline_inlineCtor_linkTest") {
     lib = "codegen/inline/inlineCtor_linkTest_lib.kt"
 }
 
+linkTest("inline_kt44764") {
+    source = "codegen/inline/kt44764_main.kt"
+    lib = "codegen/inline/kt44764_lib.kt"
```

Comment:
Minor: since the test prints a value, it makes some sense to check it here (with `goldValue`).